roles/infrastructure/defaults/main.yml

```diff
@@ -18,6 +18,17 @@ infrastructure_vnet_subnets:
       - serviceName: Microsoft.DBforPostgreSQL/flexibleServers
         name: subnet-{{ deployment_id }}-postgres-delegation
 
+# Database
+database_server_user: "user_placeholder"
```

Comment:
```suggestion
infrastructure_database_server_user: "user_placeholder"
```
I believe the role defaults should have the role name prepended for linter / consistency.